database/seeds/ContributableTableSeeder.php

```diff
@@ -20,6 +21,10 @@ public function run()
             $podcast->contributors()->attach(array_unique([rand(1,10), rand(1,10), rand(1,10)]));
         }
 
+        foreach (Category::all() as $category) {
+            $category->contributors()->attach(array_unique([rand(1,10), rand(1,10), rand(1,10)]));
+        }
+
```

Comment:
This (and everything surrounding it) is pretty opaque to me; it could probably use some comments as to what it's doing; in particular, what the three random integers represent.